src/app/devices/general/general.component.html

```diff
@@ -41,9 +41,7 @@
         <div class="flex flex-50 justify-end">
           @if (!isLoading) {
             <span class="mat-h3" data-cy="amtSku">
-              @if (amtVersion?.CIM_SoftwareIdentity?.responses[6].VersionString) {
-                {{ amtVersion?.CIM_SoftwareIdentity?.responses[6].VersionString }}
-              }
+                {{ amtBuild }}
```

Comment:
Don't we need a if condition? I don't see a look up like amtSku. Also, data-cy value should be amtBuild

Reply:
updated!